### DISCUSSION TITLE
How can I print to standard output what rules are currently enabled given my config?

### DISCUSSION THREAD

**Discussion by jorgegomezcq** [astral-sh/ruff#8724] [Q&A]
Hi,

How can I print to standard output what rules are currently enabled given my config?

The issue is that I have a consortium of docstrings enabled or disabled along with individual ones enabled/disabled.

It is too opaque and tedious for me to ultimately tell which ones are enabled or disabled.

WDYT

**Comment by zanieb** [comment]
You can sort of get this with `ruff check . --show-settings` then look at the `linter.rules.enabled` section. These use the rule names instead of the codes though.

There might be an easier way I don't know off the top of my head. If not, we should make it easier.

**Reply by zanieb** [reply]
ref #8334

**Reply by jorgegomezcq** [reply]
Hi!

 I tried `ruff check . --show-settings` already last week. Thanks for the information, though. It is much appreciated. I do agree that we should make it better.

Thanks for linking #8334 

Hopefully, I'll have time soon to investigate #8334. I'd like to add a cleaner output of currently enabled ruff rules given an arbitrary `ruff.toml`, etc.
